tpgtools/property.go

```diff
@@ -486,6 +486,11 @@ func createPropertiesFromSchema(schema *openapi.Schema, typeFetcher *TypeFetcher
 		// won't be set initially.
 		v.Title = k
 
+		if parent == nil && v.Title == "id" {
```

Comment:
Note that changing the title at this stage will make it so that the field needs to be called `other_id` in overrides, so the current `id` override doesn't apply.

Reply:
Removed the need for an override, called the resulting id resource.Name() + "Id"